dist/docker-images/ziti-controller/Dockerfile

```diff
@@ -19,6 +16,22 @@ LABEL name="openziti/ziti-controller" \
 # set up image as root, then drop privs to ziggy
 USER root
 
+# used by entrypoint.bash to source the bootstrapping script in this image
+ENV ZITI_CTRL_BOOTSTRAP_BASH=/bootstrap.bash
+
+COPY ${CONTROLLER_PACKAGE}/bootstrap.bash ${ZITI_CTRL_BOOTSTRAP_BASH}
+COPY ${CONTROLLER_PACKAGE}/entrypoint.bash /
+
+WORKDIR /ziti-controller
+RUN chown -R ziggy:ziggy /ziti-controller
+
+# configure logging
+ENV PFXLOG_NO_JSON=true
+ENV ZITI_TIME_FORMAT=utc
+
+# globally toggle bootstrapping; false preserves "thin" behavior for existing deployments like K8s
+ENV ZITI_BOOTSTRAP=false
```

Comment:
bootstrapping is disabled by default in the image to preserve existing "thin" deployments, e.g., k8s